mono/metadata/class.c

```diff
@@ -6615,10 +6615,13 @@ mono_class_from_name (MonoImage *image, const char* name_space, const char *name
 		if (res) {
 			if (!class)
 				class = search_modules (image, name_space, name);
```

Comment:
But continuing here is the desired new behavior. I want that case to reach the rest of the mono_class_from_name function, where type forwarders would be resolved. Do you see any problems with that?